### PR TITLE
Add New Post - Community Update - June 2021

### PR DESCRIPTION
**Pull request by steveyiyo**

### CODE REVIEW

**Comment by steveyiyo** [comment]
Please also check the translation of the following.

> their sponsorship ensures Rocky’s status as a first-class citizen on the Google Cloud Platform from day one, with supported images immediately available for launch.

> 他們的贊助確保了 Rocky 成為 Google Cloud Platform 上的一等公民，並立即提供支援的圖像以供發布。 

> To say that Rocky Linux 8.4 is popular would be an understatement.
> You, the community, made this happen. We can’t begin to express our gratitude for your support. Watching the community grow from a comment in a blog post in December to what it is today, with teams around the world working on documentation, development, branding, security, infrastructure, and more, has been one of the most rewarding and humbling experiences we’ve ever been a part of.

> 說 Rocky Linux 8.4 很受歡迎，其實是輕描淡寫。 
> 整個社群（包括你），讓這一切成真了！  
> 我們無法開始對您的支援表示感謝。看著社群從 12 月的一篇部落格文章中發展到現在，來自世界各地的團隊致力於文件、開發、品牌、安全及基礎建設等，一直是我們參與過最有價值及最謙卑的經歷之一。

**Comment by pwtsai** [comment]
GCP has added Rocky Linux as an option in OS list. I suggest to make a small change like: 
"他們的贊助確保了 Rocky Linux 成為提供給 Google Cloud Platform 客戶的最優質選擇，並立即新增了代表圖示。"
Please correct me if I misunderstood the original meaning.

**Comment by steveyiyo** [comment]
@pwtsai Looks good, Thanks!

File: blogs/community-update-june-2021.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+**您**，與整個社群，讓這一切成真了！我們對您的支持十分感激。
+看著社群從一篇部落格文章中的留言一路發展至今，如今來自世界各地的團隊致力於文件、開發、品牌、安全及基礎建設等，一直是我們參與過最有價值的經歷。
+
+我們很高興宣布兩個有關贊助商與合作夥伴的更新：
+
+Google 已簽約成為 RESF 的主要贊助商！Google 知曉 Rocky Linux 作為免費、開放及社群作業系統的重要性。  
+Google 提供了用於測試及驗證的資源，並確保了 Rocky 在 Google Cloud Platform 上坐擁一席之地，現在您可以立即在 GCP 中找到可用的 [鏡像](https://cloud.google.com/compute/docs/images/os-details#rocky_linux) 檔案。 
+
+與此同時，Microsoft 也簽約成為 RESF 的合作夥伴！以下內容來自他們：
```
Comment (evshary) [inline blogs/community-update-june-2021.md:31]: 以下內容來自他們： => 他們表示：

File: blogs/community-update-june-2021.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### 摘要
+
+經過七個月的努力，我們很高興的宣布 Rocky Linux 8.4 已發佈 x86-64 及 aarch64 版本！
+我們在候釋版本上取得驚人的進展，並且非常推薦您將此版本安裝於生產環境上。  
+我們的 GitHub 存儲庫中提供了 [轉換工具](https://github.com/rocky-linux/rocky-tools/tree/main/migrate2rocky)，新的 [釋出說明](https://docs.rockylinux.org/releasenotes/8.4/) 也可在我們的新文件網站上查看。
+
+在 Rocky Linux 8.4 釋出後的 72 小時內，Rocky 的資源已從 Fastly 提供的 Tier0 鏡像中被下載了近 70,000 次，其中不包含近 100 次直接從我們的鏡像中下載的次數。此外透過 Torrent 來下載的大約有 10,000 次。
+
+說 Rocky Linux 8.4 很受歡迎，其實是輕描淡寫。
```
Comment (evshary) [inline blogs/community-update-june-2021.md:21]: 用受歡迎來形容 Rocky Linux 8.4 都太過輕描淡寫了。

File: blogs/community-update-june-2021.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+> Linux 是 Azure 中成長最快的平台，如今過半數的 VM 皆為 Linux 核心。十幾年來，Microsoft 一直在投資 Linux，合作夥伴的關係更是 Microsoft 開源策略的核心支柱。
+> 我們與 RESF 及更廣泛的 Rocky Linux 生態系統合作，確保客戶有更多的選擇，並以受支持、管理及安全的方式在 Azure 上部署 Linux。 
+
+我們將在近期發布如何在 Azure 上執行 Rocky Linux 的詳細內容。
+
+## 團隊更新
+
+### 領導
+
+我們收到了一些來自社群的關於 RESF 結構的問題，雖然這些內容一直在我們的網站上可用，但保險起見，我們進行了更新。
```
Comment (evshary) [inline blogs/community-update-june-2021.md:42]: 雖然這些內容一直在我們的網站上可用，但保險起見，我們進行了更新 => 雖然這些資訊在網站上一直都能查詢到，但我們還是做了更明確的更新

File: blogs/community-update-june-2021.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+我們推出了對用戶更友好的 [文件網站](https://docs.rockylinux.org/)。  
+現在相比於以往更容易做出貢獻，我們很歡迎新內容。
+我們歡迎所有人在上面貢獻，若你有興趣，歡迎與我們聯繫。
+
+---
+
+如果你有任何的意見、問題、擔憂、建議或有意願提供協助，請發送 E-mail 至 [hello@rockylinux.org](mailto:hello@rockylinux.org).
+
+順頌時祺
```
Comment (evshary) [inline blogs/community-update-june-2021.md:59]: 這個有點翻法有點奇妙，不知道謹啟會不會比較好一些
不然乾脆不要翻

**Review by evshary — CHANGES_REQUESTED** [review]

File: blogs/community-update-june-2021.md (hunk 4 above)
Comment (steveyiyo) [reply blogs/community-update-june-2021.md:59]: Sincerely to "謹啟", Thanks!

**Comment by steveyiyo** [comment]
@evshary Please review it again, Thanks!

**Review by evshary — APPROVED** [review]

**Comment by steveyiyo** [comment]
@gnehs Ping.
If you have free time, please check and approve.
Thanks

**Review by gnehs — APPROVED** [review]

**Pull request merged**